src/main/java/uk/gov/hmcts/reform/cpo/repository/CasePaymentOrderQueryFilter.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.cpo.repository;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Builder
+@Getter
+public class CasePaymentOrderQueryFilter {
+
+    public static final String CASES_TYPE_ID="caseTypeId";
+    public static final String  ORDER_REFERENCE=  "orderReference";
```

Comment:
these 2 constants doesn't seem to belong in this class

Reply:
Should I move these ones to the CasePaymentOrderEntity ?

Reply:
DONE